docs/components.md

```diff
@@ -150,6 +150,32 @@ Kibana connects to the Elasticsearch cluster via the `elasticsearch-logging` Kub
 
 Kibana is a stateless component and therefore does not have any persistent storage requirements.
 
+#### Customization
+
+##### Kibana plug-ins
+
+Add-on functionality for Kibana is implemented with plug-in modules. Some known plug-ins for Kibana are listed here: https://www.elastic.co/guide/en/kibana/6.7/known-plugins.html.
+
+To install additional plug-ins, override the `plugins` item inside the `kibana` scope, like this:
+
+```jsonnet
+(import "../../manifests/platforms/aks.jsonnet") {
+    config:: import "kubeprod-autogen.json",
+    kibana+: {
+        plugins:: {
+            logtrail: {
+                version: "v0.1.31",
+                url: "https://github.com/sivasamyk/logtrail/releases/download/v0.1.31/logtrail-6.7.1-0.1.31.zip",
```

Comment:
I think you can use in the url `$.version`.
If not that var won't be used

Reply:
`version` is used in line 79 in the following expression:

```
                local plugins = [{name: name} + $.plugins[name] for name in std.objectFields($.plugins)],
                KIBANA_PLUGINS: std.join('\n', ["%(name)s,%(version)s,%(url)s" % plugin for plugin in plugins]),
```